File: netway/main/billing.netway.co.th/public_html/includes/modules/Hosting/rvcpanel_manage2/include/cpl-3.6/php/examples/check_status.php
<?php
include("../cpl.inc.php");
$cpl = new cPanelLicensing("__USERNAME__", "__PASSWORD__");

$ip = "__IP__";

$status = (array)$cpl->fetchLicenseRaw(array(
    "ip" => $ip
    )
);

print "The license id for the ip is: ". $status["@attributes"]["licenseid"] . "\n";
print "The status of the license is: ";
if ($status["@attributes"]["valid"] > 0) {
    print "Active";
} else {
    print "Inactive";
}
print "\n";
print "The company holding the license is: " . $status["@attributes"]["company"] . "\n";

?>

File: rvglobalsoft/main/rvglobalsoft.com/public_html/includes/modules/Hosting/rvproduct_license/include/cpl-3.6/perl/examples/Check_Status.pl
#!/usr/bin/perl

use cPanelLicensing;

my $ip = '__IPTOCHEK__';
my $licenseManager = new cPanelLicensing(user => '__YOU@HOST.COM__',
                                         pass => '__YOURPASS__');

my $liscid_ref = $licenseManager->fetchLicenseRaw(ip => $ip);
if ($liscid_ref) {
    my $valid = $liscid_ref->{'valid'};
    my $company = $liscid_ref->{'company'};
    print "The license id for ip address: ${ip} is: ${liscid}\n";
    print "The valid of the license is :" . ($valid == 1 ? 'Active' : ($valid == 0 ? 'Never Activated' : 'Expired')) . "\n";
    print "The company holding the license is : " . $company . "\n";
} else {
    print "There is no active license for ip address: ${ip}\n";
}

